packages/components/src/Select/Select.tsx

```diff
@@ -77,27 +92,25 @@ const StyledSelect = styled(ReactSelect)<IStyledSelectProps>`
   }
 
   .react-select__control--is-focused {
-    box-shadow: 0 0 0px 3px ${({ theme }) => theme.colors.yellow};
-    border: solid ${({ hideBorder }) => (hideBorder ? '0px' : '2px')};
-    ${({ theme }) => theme.colors.copy};
+    outline: 0.5px solid ${({ theme }) => theme.colors.grey600};
+    border: 1.5px solid ${({ theme }) => theme.colors.grey600};
+    box-shadow: 0 0 0 4px ${({ theme }) => theme.colors.yellow};
   }
 
-  ${({ ignoreMediaQuery, theme }) => {
-    return !ignoreMediaQuery
-      ? `@media (min-width: ${theme.grid.breakpoints.md}px) {
-        width: 344px;
-      }`
-      : ''
-  }}
+  .react-select__control--is-active {
+    box-shadow: 0 0 0 4px ${({ theme }) => theme.colors.yellow};
+  }
```

Comment:
What is this actually supposed to do that `react-select__control--is-focused` is not doing?